### PR TITLE
Edited the about.md

### PR DESCRIPTION
**Pull request by rahmot**
I edited the `about.md`. We probably need to include the program timeline on the about page, but not so sure. Do you think that is necessary?

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -3,58 +3,57 @@ id: about
 title: About the Galaxy Mentorshing Network Program
 ---
 
-Galaxy Mentoring Network (GMN) program is an initiative by the Galaxy Community to build her members in open source projects.
```
Comment (bebatut) [inline content/about.md:6]: I would keep the first line there. Just like a TLDR

File: content/about.md [hunk 2]
```diff
@@ -3,58 +3,57 @@ id: about
 title: About the Galaxy Mentorshing Network Program
```
Comment (bebatut) [inline content/about.md:3]: Mentoring? Mentorship?

File: content/about.md [hunk 3]
```diff
@@ -3,58 +3,57 @@ id: about
...
-- Mentees receive expert advice on their projects, goals, and the community.
-- Shorter learning curve through the mentorship relationship.
-- Members are empowered to acquire important social, organizational, and technical skills.
-- Mentors grow their expertise by guiding members of the community.
-- An opportunity to share experience and knowledge.
-- Learning and leadership opportunity.
+Galaxy is a sprawling ecosystem and a dispersed community. It is easy for newcomers to get overwhelmed by the many Galaxy resources and community communication channels. The Galaxy Mentor Network is established to connect newcomers with experienced community members to help prevent newcomers from getting lost.
 
-## Application Guidelines
+<div class="img-fluid mx-auto trim-p">
```
Comment (bebatut) [inline content/about.md:18]: I would remove that for now. We should avoid HTML in Markdown. There should be a way to change the class of the image directly with Markdown

File: content/about.md [hunk 4]
```diff
@@ -3,58 +3,57 @@ id: about
...
 
-### Eligibility
+![The Galaxy Community at GCC2019](assets/images/gcc2019-grouppicture.jpg)
 
-The Galaxy Community Mentorship program is open to everyone who meets the following requirements:
+<small>GCC2019 in Freiburg, Germany, photo by Bérénice Batut</small>
```
Comment (bebatut) [inline content/about.md:22]: You can maybe put that on the bottom for the page

File: content/about.md [hunk 5]
```diff
@@ -3,58 +3,57 @@ id: about
...
-- You are an active member of the community (for example, in the channels, contributing to the Galaxy GitHub repositories, events, e.t.c)
-- Willing to give detailed and timely reviews of the projects and contributions of your GMentees.
-- You are willing to meet with the GMentee on a biweekly 30 minutes call.
-- GMentors will be required to mentor for at least 8 weeks.
-- GMentors will be expected to recommend helpful projects if the GMentee has no intended projects during application.
+## Vision
+
+We hope to increase the proficiency of members in the use of Galaxy tools and servers, contributing to open source projects as well as becoming active members of any of our channels and working groups.
 
-### Matching Process:
```
Comment (bebatut) [inline content/about.md:45]: Where will be all the content you deleted there?

Comment (rahmot) [reply content/about.md:45]: Good morning! I moved the content to `application.md`.

**Comment by rahmot** [comment]
@bebatut, thanks for your guidance. I have made changes to the `about.md`.

File: content/about.md [hunk 6]
```diff
@@ -15,24 +17,18 @@ Galaxy is a global community of researchers, trainers, students, bioinformaticia
...
-## Galaxy Mentoring Network (GMN)
+## Our Vision
 
-Galaxy Mentoring Network (GMN) program is an initiative by the Galaxy Community to build her members in open source projects.
+We hope to increase the proficiency of members in the use of Galaxy tools and servers, contributing to open source projects as well as becoming active members of any of our channels and working groups.
 
-It is run on an open enrollment basis where mentees and mentors can apply at any time of need, this is so especially because we have a lot of new members who join regularly and may immediately need someone to guide them. This is a program by Galaxy members and for Galaxy members.
 
-## Vision
+## The GMN Mentoring Program
```
Comment (bebatut) [inline content/about.md:28]: Is it not redundant "Galaxy Mentoring Network Mentoring Program"?

File: content/about.md [hunk 7]
```diff
@@ -15,24 +17,18 @@ Galaxy is a global community of researchers, trainers, students, bioinformaticia
 
 Galaxy is a sprawling ecosystem and a dispersed community. It is easy for newcomers to get overwhelmed by the many Galaxy resources and community communication channels. The Galaxy Mentor Network is established to connect newcomers with experienced community members to help prevent newcomers from getting lost.
 
-<div class="img-fluid mx-auto trim-p">
-
-![The Galaxy Community at GCC2019](assets/images/gcc2019-grouppicture.jpg)
+![The Galaxy Community at GCC2019](assets/images/gcc2019-grouppicture.jpg)*GCC2019 in Freiburg, Germany, photo by Bérénice Batut*
```
[the remark this answers is not in the document]
Comment (bebatut) [reply content/about.md:20]: Sorry I meant `src/images`. And can you make the change directly in this PR?

File: content/about.md [hunk 8]
```diff
@@ -1,60 +1,55 @@
...
-- An opportunity to share experience and knowledge.
-- Learning and leadership opportunity.
+Galaxy is a global community of researchers, trainers, students, bioinformaticians, software developers, and infrastructure providers, all using the [Galaxy platform](https://galaxyproject.org/) to perform and support data-intensive research. There are hundreds (possibly thousands) of Galaxy deployments around the world, and tens of thousands of researchers and students use Galaxy every year. Hundreds of community members contribute to the project every year, through support, training, presentations, events, code, tools, testing, and documentation.
 
-## Application Guidelines
+Galaxy is a sprawling ecosystem and a dispersed community. It is easy for newcomers to get overwhelmed by the many Galaxy resources and community communication channels. The Galaxy Mentor Network is established to connect newcomers with experienced community members to help prevent newcomers from getting lost.
 
-### Eligibility
+![The Galaxy Community at GCC2019](assets/images/gcc2019-grouppicture.jpg)*GCC2019 in Freiburg, Germany, photo by Bérénice Batut*
```
Comment (bebatut) [inline content/about.md:20]:
```suggestion
![The Galaxy Community at GCC2019](/assets/images/gcc2019-grouppicture.jpg)*GCC2019 in Freiburg, Germany, photo by Bérénice Batut*
```

File: content/about.md [hunk 9]
```diff
@@ -1,60 +1,55 @@
...
-- An opportunity to share experience and knowledge.
-- Learning and leadership opportunity.
+Galaxy is a global community of researchers, trainers, students, bioinformaticians, software developers, and infrastructure providers, all using the [Galaxy platform](https://galaxyproject.org/) to perform and support data-intensive research. There are hundreds (possibly thousands) of Galaxy deployments around the world, and tens of thousands of researchers and students use Galaxy every year. Hundreds of community members contribute to the project every year, through support, training, presentations, events, code, tools, testing, and documentation.
 
-## Application Guidelines
+Galaxy is a sprawling ecosystem and a dispersed community. It is easy for newcomers to get overwhelmed by the many Galaxy resources and community communication channels. The Galaxy Mentor Network is established to connect newcomers with experienced community members to help prevent newcomers from getting lost.
 
-### Eligibility
+![The Galaxy Community at GCC2019](src\images\gcc2019-grouppicture.jpg)*GCC2019 in Freiburg, Germany, photo by Bérénice Batut*
```
Comment (bebatut) [inline content/about.md:20]:
```suggestion
![The Galaxy Community at GCC2019](/src/images/gcc2019-grouppicture.jpg)*GCC2019 in Freiburg, Germany, photo by Bérénice Batut*
```

Comment (rahmot) [reply content/about.md:20]: Oh, it is a backward slash. I pasted the copied relative path directly. Will take note henceforth.Thanks!

**Pull request merged**